Ex1/main.java

```diff
@@ -163,4 +164,47 @@ public static void saveToFile(ArrayList<Contact> phone_book)
             e.printStackTrace();
         }
     }
+
+    private static void reverseOrder(ArrayList<Contact> phone_book) {
+		// input: phone book
+            // output: phone book in reversed order
+		int half_size=phone_book.size()/2;
+		for(int i=0;i<half_size;i++)
+		{
+			Collections.swap(phone_book,i,phone_book.size()-1-i);
+
+		}
+	}
+
+	private static void sortAZ(ArrayList<Contact> phone_book) {
+		// input: phone book
+		//output: phone sorted by dictionary order
+		for(int i=0;i<phone_book.size()-1;i++)
```

Comment:
```suggestion
		for(int i=0;i<phone_book.size();i++)
```
If you start from i=0 and you iterate until i is smaller than size and not smaller\equal then -1 is not required.
example :
phone_book.size() = 10
we have contacts in indexes 0 to 9
then i should iterate from 0 to 9, ( i<10 --> i<size() )